mods/default/mapgen.lua

```diff
@@ -1527,10 +1475,8 @@ local mg_name = minetest.get_mapgen_setting("mg_name")
 if mg_name == "v6" then
 	default.register_ores()
 	default.register_mgv6_decorations()
-	minetest.register_on_generated(default.generate_nyancats)
 elseif mg_name ~= "singlenode" then
 	default.register_biomes()
 	default.register_ores()
 	default.register_decorations()
-	minetest.register_on_generated(default.generate_nyancats)
```

Comment:
In fact these other lines are useful in singlenode to allow 'place registered ores' / 'place registered decorations' so it's best to remove ```if mg_name ~= "singlenode"``` from here.
@rubenwardy